### PR TITLE
notify_template.html - include proper html tags to allow translation

### PR DESCRIPTION
**Pull request by horus68**
Formating include/language/en_us.notify_template.html
with html tags to allow translation on Crowdin.

This cause an issue on email subject filed but its solved by a related PR #1472 and #1654 that should be implemented at same time.

This PR is a recreation from #867 because I had to delete my repo.

### CODE REVIEW

File: include/language/en_us.notify_template.html [hunk 1]
```diff
@@ -43,123 +43,123 @@
...
+<p>{ASSIGNER} has assigned a Case to {ASSIGNED_USER}.</p>
 
-Subject: {CASE_SUBJECT}
-Priority: {CASE_PRIORITY}
-Status: {CASE_STATUS}
-Description: {CASE_DESCRIPTION}
+<p>Subject: {CASE_SUBJECT}</br>
+Priority: {CASE_PRIORITY}</br>
+Status: {CASE_STATUS}</br>
+Description: {CASE_DESCRIPTION}</br>
```
Comment (samus-aran) [inline include/language/en_us.notify_template.html]: missing a </p> here

File: include/language/en_us.notify_template.html [hunk 2]
```diff
@@ -169,205 +169,205 @@
...
-To: {CALL_TO}
- {ASSIGNER} has invited you to a Call
-Subject: {CALL_SUBJECT}
-Status: {CALL_STATUS}
-Start Date: {CALL_STARTDATE}
-Duration: {CALL_HOURS}h, {CALL_MINUTES}m
-Description: {CALL_DESCRIPTION}
-
-Accept this call:
+<p>To: {CALL_TO}<p>
```
Comment (samus-aran) [inline include/language/en_us.notify_template.html]: missing an end paragraph tag here (missing the /)

**Comment by horus68** [comment]
@samus-aran  html fixes done on new commit

**Comment by samus-aran** [comment]
I've referenced this comment from reviewing the latest.

https://github.com/salesagility/SuiteCRM/pull/1702#issuecomment-230848784

**Pull request closed without merging**